tests/integration/test_storage.sh

```diff
@@ -69,20 +69,20 @@ trap teardown EXIT
 # If /etc/sysconfig/docker-storage-setup is missing, atomic should create the file.
 
 rm -f /etc/sysconfig/docker-storage-setup
-OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 2>&1)
+if [ -z "$VGROUP" ];then
+   VGROUP="test-atomic-storage"
+   OUTPUT=$(${ATOMIC} storage modify --add-device $TEST_DEV_1 --vgroup $VGROUP 2>&1)
```

Comment:
"I think setting the vgroup requires a reset first."

I am still fixing the tests, however I don't see the relevance.
The volume group (VG) is being set here first time. why would it require a `reset` first ?